docs/docs/readme.md

```diff
@@ -27,66 +29,17 @@ Pomerium can be used to:
   - [Custom applications](/docs/topics/getting-users-identity.md)
 - provide a **VPN alternative**.
 
-## Architecture
+## Demo
 
-### System Level
+To make this a bit more concrete, click the image thumnail to see a short youtube demo:
 
-Pomerium sits between end users and services requiring strong authentication. After verifying identity with your identity provider (IdP), Pomerium uses a configurable policy to decide how to route your user's request and if they are authorized to access the service.
+[![demo](https://img.youtube.com/vi/ddmrkvBSO60/0.jpg)](https://www.youtube.com/watch?v=ddmrkvBSO60 "Pomerium demo")
 
-<img alt="pomerium architecture diagram" src="./img/pomerium-system-context.svg" width="65%">
+In the above video shows what the flow for what both an unauthorized and authorized user.
```

Comment:
Should be `The above video shows the flow for both an unauthorized and authorized user.`